pkg/agent/apiserver/handlers/ovsflows/handler.go

```diff
@@ -240,17 +263,14 @@ func HandleFunc(aq agentquerier.AgentQuerier) http.HandlerFunc {
 			return
 		}
 
-		err = json.NewEncoder(w).Encode(resps)
-		if err != nil {
-			w.WriteHeader(http.StatusInternalServerError)
-		}
+		encodeResp()
 	}
 }
 
 var _ common.TableOutput = new(Response)
 
 func (r Response) GetTableHeader() []string {
-	return []string{"FLOW"}
+	return []string{""}
```

Comment:
The "FLOW" doesn't provide extra info and it's strange it have this header when printing the table name only, so removed this table header.